src/app/app.module.ts

```diff
@@ -53,7 +61,7 @@ import { environment } from '../environments/environment';
     { provide: ErrorHandler, useClass: ErrorService },
     { provide: DateAdapter, useClass: MomentDateAdapter, deps: [MAT_DATE_LOCALE, MAT_MOMENT_DATE_ADAPTER_OPTIONS] },
     { provide: MAT_MOMENT_DATE_ADAPTER_OPTIONS, useValue: { useUtc: true } },
-    { provide: MAT_DATE_FORMATS, useValue: MAT_MOMENT_DATE_FORMATS }
+    { provide: MAT_DATE_FORMATS, useValue: PREFERRED_DATE_FORMATS }
```

Comment:
I also updated the display date formats for the material datepickers globally in the app (currently just the dashboard and flight form.) Angular has an [example on this config](https://material.angular.io/components/datepicker/examples).